realm/realm-library/src/androidTest/java/io/realm/RealmMigrationTests.java

```diff
@@ -1209,6 +1211,70 @@ public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
         Realm.migrateRealm(config, migration);
     }
 
+    @Test
+    public void renameAndAddField() {
+        RealmMigration migration = new RealmMigration() {
+            @Override
+            public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
+                realm.schema.get("MigrationFieldRenameAndAdd")
```

Comment:
use `getSchema()` instead of `schema`. Not sure why `schema` is package protected to be honest 😕